src/ray/core_worker/core_worker.h

```diff
@@ -14,6 +14,8 @@ namespace ray {
 /// The root class that contains all the core and language-independent functionalities
 /// of the worker. This class is supposed to be used to implement app-language (Java,
 /// Python, etc) workers.
+///
+/// Note: the constructor of CoreWorker would throw if a failure happens.
```

Comment:
Nite: put this note in the constructor? And elsewhere in the codebase, we use this format `// NOTE (author_name): ...`.

Reply:
Thanks. btw I updated the TODOs in other places as well.